client/src/app/site/pages/organization/pages/accounts/services/common/account-controller.service.ts

```diff
@@ -73,9 +73,10 @@ export class AccountControllerService extends BaseController<ViewUser, User> {
                 return {
                     id: user.id,
                     meeting_id: meeting.id,
-                    group_ids: groupIds?.includes(meeting.default_group_id)
+                    group_ids: (groupIds?.includes(meeting.default_group_id)
                         ? groupIds
                         : (groupIds ?? []).concat(meeting.default_group_id)
+                    ).filter(id => !!id)
```

Comment:
If there is a reason to always add the default group (which I guess there is) this looks like a very bad approach to "fix" this.